hudi-common/pom.xml

```diff
@@ -172,6 +172,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.ben-manes.caffeine</groupId>
```

Comment:
what this dependency used for?

Reply:
used to cache the history  schema. caffine is better than guava cache